deploy-service/common/src/main/java/com/pinterest/deployservice/handler/GoalAnalyst.java

```diff
@@ -314,7 +333,7 @@ AgentBean genUpdateBeanByReport(PingReportBean report, AgentBean agent) {
     // Generate new agent bean based on the report & current agent record,
     // This is intended to be used for deploy goal to install next stage
     AgentBean genNextStageUpdateBean(EnvironBean env, PingReportBean report, AgentBean agent) {
-        AgentBean agentBean = genUpdateBeanByReport(report, agent);
+        AgentBean agentBean = genAgentBeanByReport(report, agent);
```

Comment:
can we still call genUpdateBeanByReport() to minimize/isolate the changes. as the logic is to done mainly  inside the genUpdateBeanByReport()

Reply:
updating a diff.